### ISSUE TITLE
Infrared sensor

### ISSUE THREAD

**Issue by joheko** [AlexGustafsson/dobot-python#6]
Hey! have you used an infrared sensor?  if so does it work correctly

**Comment by AlexGustafsson** [comment]
No, I haven't. I haven't seen any mentioning of an infrared sensor in the protocol documentation. Perhaps it's a newer addition? Feel free to open a PR if you end up investigating it further.

**Comment by joheko** [comment]
I meant this.
![1B8982BA-4B66-407B-9502-527E593B5CC8](https://user-images.githubusercontent.com/107461717/183250709-fe8bfe6d-a9be-4ab6-942e-b1eb704e2add.jpeg)

**Comment by AlexGustafsson** [comment]
Oh, my bad, it's there! It's implemented according to the specification, so there's a good chance that it works. I haven't tried it though.
